src/Components/UI/Modal/EditNoteModal.jsx

```diff
@@ -59,7 +59,7 @@ const EditNoteModal = () => {
   const initialNoteDetails = {
     title: "",
     pinned: false,
-    tags: [],
+    labels: [],
     date: new Date().toLocaleDateString(),
   };
```

Comment:
Suggestion: you could directly pass initialNoteDetails with you noteProvider to avoid repetition of code.

Reply:
Thank you will fix this.